### ISSUE TITLE
Add advice for standard min diff multipliers in calculator

### ISSUE THREAD

**Issue by aaronyan** [Automattic/abacus#762]
While reading into how other platforms deal with **minimum practical difference**, I learned the VWO uses [threshold for caring](https://help.vwo.com/hc/en-us/articles/360033471774-What-is-a-Winner-in-VWO-#:~:text=The%20threshold%20of%20caring%20is,Smart%20Decision%2FWinner%20or%20not.). And they provide a recommended multiplier of `0.001` for revenue and `0.0075` for non-revenue goals.

What do you think about adding a **tooltip** to minimum practical difference that reads: 
- For conversion metrics: "recommended value: current conversion rate * 100 * `0.005`"
- For revenue metrics: "recommended value: current ARPU * `0.005`"

We could also instead have a tooltip with a link to the example min diff calculation in the FG!

**Comment by aaronyan** [comment]
What do y'all think about t-shirt sizes for min diffs? If we know a relative range of baseline conversion rate or ARPU, then we can simply apply the multiplier for experimenters. What if we had these as dropdown options:

Conversion metrics:
- Custom
- XS (0 to 2%) -> min diff of `0.01`
- S (2-10%) -> min diff of `0.05`
- M (10-30%) -> min diff of `0.15`
- L (30-60%) -> min diff of `0.3`
- XL (60-100%) -> min diff of `0.5`

Revenue metrics:
- Custom
- XS ($0-$0.01) -> min diff of `0.005`
- S ($0.01-$0.05) -> min diff of `0.025`
- M ($0.05-$0.10) -> min diff of `0.05`
- L ($0.10-$0.30) -> min diff of `0.15`
- XL (>$0.50) -> min diff of `0.25`

**Comment by jessie-ross** [comment]
I think we should be careful about taking things from Experimentation Platforms As A Service, they are incentivised to make what sells which especially includes shortcuts like a default multiplier.

We should rather be focused on what internal experimentation platforms do, we could even organise a chat with experimentation platform developers from other companies.

> threshold for caring
I don't mind calling it something different but I don't want to dumb it down too much (by providing standard values), I would like experimenters to learn the concept and understand what they are working with, I think A12s are very capable. I think the calculator helps a lot as it removes the "unit" complication of the matter.

**Comment by aaronyan** [comment]
> I think we should be careful about taking things from Experimentation Platforms As A Service, they are incentivised to make what sells which especially includes shortcuts like a default multiplier.

That's a fair caution; their incentives may not be aligned with ours. I will keep this in mind, but I think those services also offer a lot of value that we can learn from! We are bumping into a lot of similar issues these other services face, and I think it's because we share a similar audience to the explats-as-a-service.

I'd wager internal experimentation at other companies that can afford to build their own platforms can also afford data analysts/ scientists to run their experiments. Since we're democratizing experimentation to anyone at the company, I think our audience is closer to the explats-as-a-service rather than internal experimentation platforms.

> We should rather be focused on what internal experimentation platforms do, we could even organise a chat with experimentation platform developers from other companies.

I would love that, and I would be very interested in learning from developers from other companies with internal experimentation platforms. Haha maybe I could try sending out some emails or messages on LinkedIn? What other ways would be a good way to organize something like this?

**Comment by aaronyan** [comment]
> I don't mind calling it something different but I don't want to dumb it down too much (by providing standard values), I would like experimenters to learn the concept and understand what they are working with, I think A12s are very capable. I think the calculator helps a lot as it removes the "unit" complication of the matter.

+1 to helping experimenters understand the concept! From my experience, a12s are open to learning, but they're not willing to take the steps to determine a min diff value for their metrics. I think standard multipliers and/or t-shirt sizes can help protect experimenters, even though they are shortcuts.

Standard multipliers convert min diff from an absolute value calculation to a relative one. A multiplier of `0.005` for conversion metrics says that less than 1/2 a percentage lift in the baseline would not be practically meaningful. I think it's really valuable to help experimenters reach a closer ballpark min diff, and most folks could agree a 1/2 a percentage lift is not practically meaningful in most contexts.

I get that we don't want to promote or encourage poor experimentation practices. However, by only having a custom option and allowing experimenters to ignore min diff, I fear we are passively condoning bad practices right now. We could require a link to a funnel or query for each min diff in Abacus. I think the cost of this would be too high for experimenters, but it is an alternative direction to tackling the min diff problem.

The min diff problem, in my experience:

- Experimenters tend to ignore min diff and put values from previous experiments
- When taught how to use min diff in experiment reviews, very few take the time to create a funnel and change the values
- Large min diffs are more misleading than standard min diff values (t-shirt sizes have a max `0.5 pp` and `$0.001`), because Abacus could recommend "no difference" for large min diff metrics
- There are still lots of questions around min diff, even from experienced experimenters

**Comment by razvanpapadopol** [comment]
I don't really see the value of using predefined values (t-shirt sizes) because each experiment may have an exposure event at the very beginning of the funnel or "closer" to where we evaluate conversions. So the range of baseline conversion rate / ARPU can get super wide. At the same time, we should encourage a12s to get comfortable with the absolute impact (monthly numbers) from the design phase so they will be prepared when having to interpret and share the results.

But I'm perfectly ok with using a multiplier applied to the current conversion rate / ARPU. It can be the left edge of our originally envisioned [slider-with-input-field](https://mui.com/material-ui/react-slider/#slider-with-input-field) 😁 

We could even go one step forward and add the value as a default in the min practical difference field as soon as the other 2 fields are filled in. As an experimenter, it's quite demoralizing to start from a blank canvas so any placeholder, tooltip, or template may help.

**Comment by aaronyan** [comment]
> I don't really see the value of using predefined values (t-shirt sizes) because each experiment may have an exposure event at the very beginning of the funnel or "closer" to where we evaluate conversions. So the range of baseline conversion rate / ARPU can get super wide.

Yeah, that's true. With each experiment, the baseline conversion could change dramatically based on the start of the experiment experience/ attribution window. If an experimenter doesn't know their baseline rate, they won't know what t-shirt size to pick for their metrics.

I think experimenters would still benefit, because the t-shirt sizes provide an upper-bound to selectable min diffs of `0.5 pp` and `$0.003`. I think underestimating min diff is better than overestimating min diff. There might be less complicated ways to achieve a suggested upper-bound for min diffs though. :)

> At the same time, we should encourage a12s to get comfortable with the absolute impact (monthly numbers) from the design phase so they will be prepared when having to interpret and share the results.

Yeah, I like that the min diff exercise prepares experimenters for how to interpret their results later on. Using absolute impact in the design and analysis phases is a really nice touch!

> We could even go one step forward and add the value as a default in the min practical difference field as soon as the other 2 fields are filled in. As an experimenter, it's quite demoralizing to start from a blank canvas so any placeholder, tooltip, or template may help.

Ooo I like the idea of adding some guidance! What about adding some text in the lift section of the calculator like this?

<img width="995" alt="Screen Shot 2022-08-18 at 10 09 12 AM" src="https://user-images.githubusercontent.com/4505888/185454495-79875106-bdeb-4a9c-847c-c5fcdf746842.png">

**Comment by aaronyan** [comment]
I don't think we will pursue the idea of standard min diff multipliers in the calculator, so we can close this!
